addons/move-to-top-layer/userscript.js

```diff
@@ -0,0 +1,27 @@
+// Welcome to Move to Top Layer code!
+// Code was written by Norbiros
+//
+// If you find any bugs, or you want to change anything
+// create an issue on the Scratch Addon github repository (or create a PR)!
+
+export default async function ({ addon, global, console, msg }) {  
+  const vm = addon.tab.traps.vm;
+
+  // look for sprite list
+  let spriteList = document.getElementsByClassName("sprite-selector_items-wrapper_4bcOj box_box_2jjDp");
```

Comment:
waitForElement should be used here instead. This will not work every time, and will always stop working after the user sees the project page and comes back inside.

Reply:
Fixed by my commit